test/integration/backward_compatible/parallel/ssl/ClientSSLTest.js

```diff
@@ -41,8 +43,12 @@ describe('ClientSSLTest', function () {
         return RC.terminateCluster(cluster.id);
```

Comment:
My idea was since we call shutdownAll in `after` we don't need to worry about what is done is `afterEach` blocks. We may use it just for convenience, but being sure that it won't cause any problems in tests is hard sometimes. Maybe the test author wanted to shutdown only some subset of clients, clusters etc. That's why I didn't change all of them. Let me check the logic and change it

Reply:
I think it is perfectly fine to have it just in `afterEach`

Reply:
For this case yes you are right deleted `after`